README.md

```diff
@@ -28,8 +28,22 @@ sudo apt install python3-pydbus
 
 ## Installation
 
-The extension is not yet signed by Mozilla and hence can only be added
-as temporary extension. For that, perform the following steps:
+### Firefox: via addons.mozilla.org
+
+You can get a signed version of the browser extension from AMO (the Mozilla addons marketplace).
+As this only covers the browser part, the host tooling still needs to be installed manually:
+
+1. clone this repository
+2. run `make local-install-firefox`
+6. Enable "Access your data for https://login.microsoftonline.com" under the extension's permissions
```

Comment:
1, 2, ... 6? Markdown papers over this, but as you are already on it..

Also, I would move the native installation even before the extension installation so that the latter works immediately.